### ISSUE TITLE
Router 3.0.0-beta.2 history not working

### ISSUE THREAD

**Issue by helzgate** [angular/angular#10386]
**I'm submitting a ...**
```
[x ] bug report
```

**Current behavior**
When clicking back, the page returns to the previous page but the URL in the browser remains unchanged.  I also tried the latest code from master as of July 29th and I still get the same issue.  

**Please tell us about your environment:**
running on OS X El Capitan

* **Angular version:** 
 ang2.0 rc4
 router 3.0 beta 2

* **Browser:**
 Chrome 51.0.2704.106
 Chrome Canary
 Firefox 47.0
 Safari 9.1
 IE 10 on Win7
 IE 11 on WIn8.1

**Comment by brandonroberts** [comment]
Will you reproduce in a plunker?

**Comment by helzgate** [comment]
Plunker isn't that user friendly when it comes to adding the most recent packages.  i don't see ang2 rc4 as an option and I don't have time figure it out.

**Comment by helzgate** [comment]
Found out what it was...I'm using the angular2-webpack-starter and I added the following into my `app.routes.ts` file because I thought i was supposed to (even though I have no clue what it does).  I then imported that into my bootstrap:

    export const appRouterProviders = [
        provideRouter(routes)
    ];

I ended up removing the above from my `app.routes.ts` file and no longer manually import the routes into the bootstrap.  I now using the same exact same method they show in the webpack starter and now my back-clicks work fine.
